pheps/phep-0001.md

```diff
@@ -0,0 +1,287 @@
+
+If a PHEP requires additional design and interface feedback before committing to long term stability, it must remain in "Draft". Alternatively, its scope can be reduced (e.g. by deferring some features to later PHEPs).
+
+If no progress is being made on the PHEP, the editor should consult the author. The author may resume work on the PHEP, [transfer it to another author](transferring-phep-ownership), or withdraw it. If the author takes none of these actions, the editor may choose to bring the PHEP to the community for approval or rejection in its current form.
+
+If a PHEP fails to find consensus, the author may choose to modify it for reconsideration in the future. The PHEP may also be "Withdrawn" if the PHEP author themself has decided that the PHEP is actually a bad idea, or has accepted that a competing proposal is a better alternative. A PHEP can also be "Rejected" if it fails to find consensus and the author is not willing or able to modify it. Perhaps after all is said and done it was not a good idea. It is still important to have a record of this fact.
+
+When a PHEP changes state, the PHEP text must be updated accordingly. In addition to updating the Status field, at the very least the Resolution header should be added with a direct link to the minutes of the telecons and meetings where a decision on the PHEP was made.
+
+Final PHEPs can also be replaced by a different PHEP, rendering the original obsolete.
+
+The possible paths of the status of PHEPs are as follows:
+
+![PHEP status flow diagram](phep-0001/process_flow.svg)
+
+While not shown in the diagram, "Accepted" PHEPs may technically move to "Rejected" or "Withdrawn" even after acceptance. This will only occur if the implementation process reveals fundamental flaws in the design that were not noticed prior to acceptance of the PHEP.
+
+Upon reaching "Final", "Withdrawn", or "Rejected" status, the editors update the preamble and merge the PR into `main`.
+
+## PHEP Maintenance
+<a name="phep-maintenance"></a>
+PHEPs are no longer substantially modified after they have reached the Accepted, Final, Rejected, Withdrawn, or Replaced state. Once resolution is reached, a PHEP is considered a historical document rather than a living specification and the PHEP number is a permanent identifier for the document.
+
+If changes based on implementation experience and user feedback are made to PHEPs while in the Accepted state, they should be noted in the PHEP, such that the PHEP accurately describes the implementation at the point where it is marked Final.
+
+Rejected and Withdrawn PHEPs must not be resurrected but may be used as the basis for a new PHEP.
+
+Changes to PHEPs in the Final state are limited to the preamble, and only in the event the PHEP is replaced. Final PHEPs are immutable and must be replaced by new PHEPs if necessary, rather than updated. The scope of a PHEP should be carefully considered in this light.
```

Comment:
We allow amendments to PLEPs at PlasmaPy, and use versioned records on Zenodo to keep track of them.  I'd be inclined to allow PHEPs to be amended too since that would be a lot less work than submitting an entirely new PHEP.

Reply:
I feel pretty strongly about this; it's much more clear to say "PHEP-x" than "PHEP-x version y" and I've been bitten by rolling updates. Plus then we'd need to document the update process. Nothing wrong with submitting a new one that starts with copy-paste of the old. Open to other perspectives here.

There's also the fact that if you're bringing the whole thing back in, everything is open for discussion. I think that's a good thing--it encourages re-evaluation if we reach the point where something is important enough to revisit, and it discourages "I'm going to go through this process to fix a typo that doesn't hurt anything." It would tend to discourage very small but potentially useful changes, though.

Reply:
Yeah @namurphy can you think of a concrete example or two of when a PLEP was amended? I tend to agree with @jtniehof on this one, although discouraging small but useful changes is something to consider... I could be swayed if there was a compelling PLEP amendment.

It adds a level of seriousness when you know you have to get the PHEP right on the first try though, which I like. Plus "PHEP-x" really does feel a lot clearer than "PHEP-x version y". And getting into the weeds of an update process sounds not fun. _And_ bringing the whole thing back in for discussion helps combat inertia. Idk, I'm compelled by Jon's reasoning here.

Reply:
Whichever way we fall on this, I'll be sure to capture explicit rationale in the PHEP (maybe in rejected ideas)

Reply:
@namurphy friendly reminder we're awaiting your response here.